test/Microsoft.AspNet.Authentication.Test/OpenIdConnect/OpenIdConnectHandlerTests.cs

```diff
@@ -443,12 +275,12 @@ private static TestServer CreateServer(IOptions<OpenIdConnectAuthenticationOptio
             );
         }
 
-        private static TestServer CreateServer(CustomConfigureOptions configureOptions, ILoggerFactory loggerFactory)
+        private static TestServer CreateServer(CustomConfigureOptions configureOptions, IUrlEncoder encoder, ILoggerFactory loggerFactory, OpenIdConnectAuthenticationHandler handler = null)
         {
             return TestServer.Create(
                 app =>
                 {
-                    app.UseCustomOpenIdConnectAuthentication(configureOptions, loggerFactory);
+                    app.UseCustomOpenIdConnectAuthentication(configureOptions, encoder, loggerFactory, handler);
```

Comment:
This should be configuring the test server exactly the same as real apps, just replacing the services with test services so the middleware picks up the correct encoder.  This doesn't look like the right direction for tests to be going